### ISSUE TITLE
Recordings

### ISSUE THREAD

**Issue by skillexia** [eyeson-team/eyeson-php#45]
Greetings. I am trying to get a session's recording. My understanding is that it's possible by default and the session's organizer needs to start it which I did for my test.

Here is my PHP code:

$eyeson->addWebhook('https://demo.xxxxxx.com/admin/meetings/recordings',  'recording_update');

and in admin/meetings:

    public function recordings() {
 
        $post = $this->input->post();  // using codeIgniter this is like $_POST
log_message('special','admin/c/meetings/recordings='. print_r($post,true));  // my own logging which is empty
        $this->output->set_header('HTTP/1.0 200 OK');

        exit; 
    }

So... what's wrong? THanks

Mike

**Comment by unused** [comment]
Please ensure your receiving endpoint is public available and does accept a HTTP POST request that will contain your recording details. [You can check the last response status code and timestamp](https://eyeson-team.github.io/api/api-reference/#register-webhooks) using the `/webhooks` GET endpoint. Additionally ensure that you set the webhook only once, therefor it is very likely you do not need to setup this process in any framework. Do not hesitate to leave a comment if you have further questions. Best wishes!

**Comment by skillexia** [comment]
This is not clear to me. Unlike Stripe or other main API providers you
don't provide examples in php, ruby, curl etc and that's why I am asking
one in PHP. I have also tried to just get the access_code with no more
success. Is the a communauty. of devs that can help since, I. am sorry to
say, your own explanations are cryptic.

**Comment by unused** [comment]
Thank you for the feedback, I'm sorry to hear that you have troubles with the documentation. The interface to control eyeson meetings is kept very simple and sticks to web standards so you can use any programming language without further packages. However there are some libraries to ease the quickstart.

A proper way to access the JSON formatted body of a POST request in PHP is using `json_decode(file_get_contents('php://input'), true)`.

**Comment by skillexia** [comment]
Thanks Christoph.

I had figured out the json_decode... finally.

I guest you post this code somewhere:

        http_response_code(200); // PHP 5.4 or greater

        $input = @file_get_contents("php://input");
        $recording = (array) json_decode($input);

 I had added this code after creating a room:

       $eyeson->addWebhook(HTTPS. SERVER_NAME.'/admin/meetings/recordings',
 'recording_update');

However, at some point I removed it but the webhook still worked. Is this
webhook needed each time or once per API_KEY?

One more question:

 I set the locale to french for the session's organizer. However, invited
guests are set in English. Is there a way to force french from the "invite
link'?



Thanks
